Servers/easily-check-a-database-connection-from-a-winows-server.md

```diff
@@ -0,0 +1,20 @@
+{{{
+  "title": "Easily Check a Database Connection from any Windows Server",
+  "date": "3-20-2015",
+  "author": "Jake Malmad",
+  "attachments": [],
+  "contentIsHTML": false
+}}}
+
+###OVERVIEW
+A common troubleshooting step in database backed applications is making certain that communication is possible from the web or application server directly to the database. This article outlines a fast and easy way to verify connectivity from a Windows Server to a Microsoft SQL database.
+
+###Detailed Steps
+1. Create a new text folder on the server by right click-> New -> "New Text Document". You may want to name it similar to the database to be tested, in this instance it is simply called "test.txt"
+1. Change the file extension to .udl. You may need to enable "Show File Extensions" in the view pane
+![Show File Extensions](../images/easily-check-a-database-connection-from-a-windows-server-1.PNG)
```

Comment:
@jmalmadclc just need to add a couple of line breaks between images and text so that the image will render on a new line.